Tests/SwiftProtobufTests/Test_Unknown_proto2.swift

```diff
@@ -133,48 +133,48 @@ class Test_Unknown_proto2: XCTestCase, PBTestHelpers {
         XCTAssertEqual(collector.collected, [Data(bytes: bytes)], line: line)
     }
 
-    func test_MessageNoStorageClass() {
+    func test_MessageNoStorageClass() throws {
         // Reusing message class from unittest_swift_extension.proto that were crafted
         // for forcing/avoiding _StorageClass usage.
         var msg1 = ProtobufUnittest_Extend_MsgNoStorage()
         assertUnknownFields(msg1, [])
 
-        msg1.unknownFields.append(protobufData: Data(bytes: [1, 2]))
-        assertUnknownFields(msg1, [1, 2])
+        try msg1.merge(serializedData: Data(bytes: [24, 1]))  // Field 3, varint
+        assertUnknownFields(msg1, [24, 1])
```

Comment:
Thanks!  Since you're removing the only use of unknownFields.append outside of the library, would you mind marking the `append` method `internal`?